components/content-picker/index.js

```diff
@@ -142,7 +142,7 @@ const ContentPicker = ({
 				)
 			)}
 
-			{Boolean(content?.length) > 0 && (
+			{Boolean(content?.length) && (
```

Comment:
@fabiankaegy this feels odd since it's casting a Boolean and then comparing it to a number. Given `Boolean(0)` will evaluate to `false` I thought better to condense this and avoid strange coercions

Reply:
👍